config/api_key_env.yml

```diff
@@ -0,0 +1,2 @@
+API_KEY: "YGUmMw55U3s1mArR6tdoAcyWVizGAIAub2kxDtp_Xic"
```

Comment:
Don't put this file in version control. put this file in gitignore because API keys are not meant to be published publically